jsons/Nations.json

```diff
@@ -637,6 +636,39 @@
+		Aeacides's son Pyrrhus came to the throne in 295 BC. Pyrrhus, being a skillful general, was encouraged to aid the Greeks of Tarentum and decided to initiate a major offensive in the Italian peninsula and Sicily. Due to its superior martial abilities, the Epirote army defeated the Romans in the Battle of Heraclea (280 BC). Subsequently, Pyrrhus's forces nearly reached the outskirts of Rome, but had to retreat to avoid an unequal conflict with a more numerous Roman army. The following year, Pyrrhus invaded Apulia (279 BC) and the two armies met in the Battle of Asculum where the Epirotes won the eponymous Pyrrhic victory, at a high cost.
+		In 277 BC, Pyrrhus captured the Carthaginian fortress in Eryx, Sicily. This prompted the rest of the Carthaginian-controlled cities to defect to Pyrrhus. Meanwhile, he had begun to display despotic behavior towards the Sicilian Greeks and soon Sicilian opinion became inflamed against him. Though he defeated the Carthaginians in battle, he was forced to abandon Sicily.
+		Pyrrhus's Italian campaign came to an end following the inconclusive Battle of Beneventum (275 BC). 
+		Having lost the vast majority of his army, he decided to return to Epirus, which finally resulted in the loss of all his Italian holdings. Because of his costly victories, the term 'Pyrrhic victory' is often used for a victory with devastating cost to the victor.",
+		"startIntroPart2": "Now is your time to take controll of that powerfull civilization, and shows the world that Pyrrhic Victory, means doom to your enemys, not to your own army.",
+		"declaringWar": "There can be only one king. Prepare for battle!",
+		"attacked": "I think my emissary must have been late, I already declared war on you!",
+		"defeated":  "You were a worthy opponent. The gods will welcome me!",
+		"introduction":  "Welcome to Epirus, friend. I am king Pyrrhus. Are you another expenda... I mean Ally... or perhaps a next target of my army?",
+		"neutralHello": "Greeting",
+		"neutralLetsHearIt": ["Yes.","Let's discuss."],
+		"neutralNo": ["No way.","Never!"],
+		"neutralYes": ["Very good.","Of course, yes.","I accept."],
+		"hateHello": "Go on!",
+		"hateLetsHearIt": ["Speak!"],
+		"hateNo": ["I can't accept!","You need to try harder."],
+		"hateYes": ["Let it be.","This offer is too good to reject."],
+		"afterPeace": "Hail, great lord. This was indeed a good fight.",
+		"tradeRequest": "That should do.",
+		"innerColor": [	232, 170, 158],
+		"outerColor": [	94, 50, 0],
+		"uniqueName": "Pyrrhic Victory",
+		"uniqueText": "Can't train Settlers. All military units have + 50% more strength when attacking, but suffer -25% penalty when defending. Gain Free Worker whenever you conquer a new city. +50% Production when constructing military units.",
+		"uniques": ["[Military] units gain the [Pyrrhic Victory] promotion","[+50]% Production when constructing [Military] units [in all cities]", "Cannot build [Settler] units"],
```

Comment:
The unique text and uniques of this nation do not seem to match?

Reply:
Maybe you misread? Looks same to me.

Reply:
Looked into it a bit further, turns out that the Pyrrhic Vcitory promotion provides the exact bonuses the uniqueText says it does

Reply:
Then I guess that it is resolved.